diagnostic.md

```diff
@@ -5,20 +5,25 @@ Record your responses inside the fenced code blocks below each question.
 1.  Give an example of a visual hierarchy that could be modeled with components.
 
     ```md
-    <!-- your response here -->
+    For data down the information of the model goes to the component and then it is sent to the template.
+    Say you have information on daily chores and you want to display the information as a list.  You can use the information from the route and pass it to a component to render a list of chores.
+    But when it bubbles up the information stays in component unless you send the action out to the route.
```

Comment:
looks good.  Other UI elements could be a navbar w/ buttons, or the sidebar for a blog w/ comments.